### ISSUE TITLE
BUG: data path gets mangled with `.` in filename

### ISSUE THREAD

**Issue by seth127** [metrumresearchgroup/babylon#178]
There seems to be a catch-22 issue when there is a `.` in the filename of a `.ctl` file.

# TL;DR

I'm pretty sure that what's happening is that, whatever triggers it to add a `../` to `$DATA` when the file extension is something other than `.ctl` is getting triggered when there's an extra `.` in the filename like `1001.1.ctl`. 

This is a common pattern, for instance for Bayesian chains where the scientist would have a model `1001.ctl` and then run chains named `1001.1.ctl`, `1001.2.ctl`, etc.

I couldn't find the part of the code that manipulates of the `$DATA` but that's my guess.

Probably related: when I wrangle the `$DATA` line to get `bbi nonmem run sge 1001.1.ctl` to run, it creates an output folder called `1001` instead of `1001.1`, which I'm sure would cause other issues in real life.

So, it seems that we're parsing the filenames incorrectly somewhere, but I'm not sure where yet.

# The evidence

In this example:
```
/data/home/sethg/bbi_test_models/janelle/tranNM_final.csv # the data file
/data/home/sethg/bbi_test_models/janelle/1001.1.ctl     # the model file
```

When I run this I get the following
```
$ /data/apps/bbi nonmem run sge janelle/1001.1.ctl
INFO[0000] Successfully loaded default configuration from /data/home/sethg/bbi_test_models/babylon.yaml
INFO[0000] Beginning Local Path
FATA[0000] An error occurred during model processing: unable to open datafile at /data/home/sethg/bbi_test_models/tranNM_final.csv referenced in modelfile janelle/1001.1.ctl. Error details are open /data/home/sethg/bbi_test_models/tranNM_final.csv: no such file or directory
```
_note:_ the datafile location is one directory too high.

But when I manually delete a `../` from the `$DATA` line in the control stream then it dies with a different error, which I'll copy into here once I reproduce it.

Also, the thing above about the output folder being named `1001` instead of `1001.1` is probably the biggest smoking gun when I think about it.
